codalab/worker/docker_image_manager.py

```diff
@@ -243,10 +243,18 @@ def image_availability_state(image_spec, success_message, failure_message):
                 def download():
                     logger.debug('Downloading Docker image %s', image_spec)
                     try:
-                        self._docker.images.pull(image_spec)
+                        self.generator = self._docker.api.pull(image_spec, stream=True, decode=True)
+                        for line in self.generator:
+                            self._downloading[image_spec]['status'] = ''
+                            if line['status'] == 'Downloading':
+                                self._downloading[image_spec]['status'] = ' (%d%%)' % (
```

Comment:
Let's move the space to worker_run_state.py instead
```suggestion
                                self._downloading[image_spec]['status'] = '(%d%%)' % (
```